src/lib/webpack/webpack-client-config.js

```diff
@@ -10,14 +10,19 @@ import PushManifestPlugin from './push-manifest';
 import baseConfig from './webpack-base-config';
 
 function clientConfig(env) {
-	const { isProd, source, src } = env;
+	const { isProd, source, src /*, port? */ } = env;
 
-	return {
-		entry: {
-			bundle: resolve(__dirname, './../entry'),
-			polyfills: resolve(__dirname, './polyfills')
-		},
+	let entry = {
+		bundle: resolve(__dirname, './../entry'),
+		polyfills: resolve(__dirname, './polyfills')
+	};
 
+	if (!isProd) {
+		entry['hmr'] = `webpack-dev-server/client?http://localhost:${process.env.PORT || env.port || 8080}`;
```

Comment:
This is a temporary fix with with we can go for now. But I guess the HMR failure is related to webpack-dev-server or html-webpack-plugin.

Once it is fixed in the upstream, we can clean this off.

Reply:
It's webpack-dev-server.